frontend/src/pages/About.tsx

```diff
@@ -1,11 +1,49 @@
 import React from 'react';
 import {NavLink} from "react-router-dom";
+import styled from "styled-components";
+
 
 export default function About() {
     return (
         <nav>
-            <p>In here you can administrate the Sidekick Horses.</p>
-            <NavLink to="/members">Go to the Members page</NavLink><br />
+            <StyledDiv>
+                <img src={process.env.PUBLIC_URL + '/p7.jpg'} alt="Logo von Pferden bw" width="850"/>
+            </StyledDiv>
```

Comment:
Vereinen zu einem einzelnen Element, css statt `width=850`, statt width in px lieber z.B. `80vw` für 80% der Bildschirmbreite